src/redux/books/bookSlice.js

```diff
@@ -3,14 +3,31 @@ import { createSlice } from '@reduxjs/toolkit';
 // Create a slice of state
 const bookSlice = createSlice({
   name: 'books',
-  initialState: {
-    books: [],
-  },
+  initialState: [
+    {
+      id: 'item1',
+      title: 'The Great Gatsby',
+      author: 'John Smith',
+      category: 'Fiction',
+    },
+    {
+      id: 'item2',
+      title: 'Anna Karenina',
+      author: 'Leo Tolstoy',
+      category: 'Fiction',
+    },
+    {
+      id: 'item3',
+      title: 'The Selfish Gene',
+      author: 'Richard Dawkins',
+      category: 'Nonfiction',
+    },
+  ],
   reducers: {
-    addBook: (state, action) => {
-      state.books.push(action.payload);
+    addBook: (state, { payload }) => {
+      state.push(payload);
```

Comment:
- [Optional] Great Job for adding the add book reducer, I think it would be better if you used nonmutable array methods to improve the performance 👍
`state.books = [...state.books, action.payload]`